api/scripts/run-docker.sh

```diff
@@ -28,6 +28,9 @@ function serve() {
              ${STATSD_HOST:+--statsd-prefix $STATSD_PREFIX} \
              app.wsgi
 }
+function run_task_processor() {
```

Comment:
It feels a little odd that this sits in `run-docker.sh` and is not used by any docker file? Maybe a makefile is a better place for this?

Reply:
We should also update the `docker-compose.yml` file in the root of this repository.

Reply:
Fair point, updated.